src/amqp10/encoder.ts

```diff
@@ -41,6 +50,41 @@ function lengthOfProperties(properties?: MessageProperties) {
   return 3 + 1 + 4 + 4 + getPropertySize(properties)
 }
 
+function lengthOfApplicationProperties(applicationProperties: MessageApplicationPropertiesList) {
+  if (!applicationProperties.length) {
+    return 0
+  }
+
+  // var size = DescribedFormatCode.Size
+  // size += sizeof(byte) //FormatCode.List32
+  // size += sizeof(uint) // field numbers
+  // size += sizeof(uint) // PropertySize
+  // size += MapSize()
+  // return size
+  return 3 + 1 + 4 + 4 + getApplicationPropertySize(applicationProperties)
```

Comment:
this `3 + 1 + 4 + 4 + something` is starting to appear in multiple places, should we consider using a const?